doc/lsst.pipe.base/command-line-task-data-repo-howto.rst

```diff
@@ -0,0 +1,136 @@
+.. _command-line-task-data-repo-howto:
+
+#################################################################
+Using Butler data repositories and reruns with command-line tasks
+#################################################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` use Butler data repositories for reading and writing data.
+Butler data repositories are specially-configured directories on a filesystem.
```

Comment:
To second KT, you should think of users as giving the butler a URI, that happens currently to be a directory.   The point of the butler is that it handles getting and saving data objects for you and the details are someone else's problem.

Incidentally, we need to explain somewhere the difference between the butler and the actual I/O;  I hear people complaining that "it's slow to read afwTables using the butler" -- they mean that the afwTable I/O code is slow, and that's nothing to do with the butler.